src/Models/Services/Utils.ts

```diff
@@ -54,6 +55,10 @@ export const validate3DConfigWithSchema = (
 };
 
 export const createGUID = (isWithDashes = false) => {
+    // switch to using a library when we don't need the dashes, but fallback to the original approach when we need it.
+    if (!isWithDashes) {
```

Comment:
When will we not need the dashes? Why not call v4 when we need a non-dash guid instead of modifying this method?

Reply:
By default we don't use dashes. I think the only place we use dashes is on one spot taht calls an API.